packages/agents/sequencer/src/bindings/server/index.ts

```diff
@@ -188,6 +194,21 @@ export const bindServer = async (queueName: string, channel: Broker.Channel): Pr
     async (req, res) => api.auth.admin(req.body, res, api.post.clearCache),
   );
 
+  server.post<{ Body: RouterPingRequest }>(
+    "/router-ping",
+    { schema: { body: RouterPingRequestSchema } },
+    async (req, res) => {
+      const { router, timestamp, signed } = req.body;
+      const signerAddress = verifyMessage(`${RouterPingMessage}-${timestamp}`, signed);
+      if (router.toLowerCase() == signerAddress.toLowerCase()) {
+        await cache.routers.setLastActive(router);
+        return res.status(200).send({ message: "OK" });
```

Comment:
Should we respond to the router with the current data in the cache ?

Reply:
I think yeah. It should be fine even if the cache cleared cause it's being updated in the next cycle.